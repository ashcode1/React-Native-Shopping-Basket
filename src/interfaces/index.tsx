// Product line item
export interface Item {
  id: number;
  name: string;
  img: string;
  colour: string;
  price: number;
  quantity: number;
  addOnPress?: () => void;
  subtractOnPress?: () => void;
  removeAllOnPress?: () => void;
}

// Colour filter picker item
export interface SelectedColor {
  label: string;
  value: string;
}
