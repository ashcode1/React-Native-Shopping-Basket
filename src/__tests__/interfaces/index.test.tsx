import * as interfacesIndex from '../../interfaces/index';

describe('Interfaces Index file', () => {
  it('Remains Unchanged', () => {
    expect(interfacesIndex).toMatchSnapshot();
  });
});
