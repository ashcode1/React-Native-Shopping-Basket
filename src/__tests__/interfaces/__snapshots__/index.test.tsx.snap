// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interfaces Index file Remains Unchanged 1`] = `Object {}`;
